### PR TITLE
[SecurityBundle] test legacy CSRF configuration options

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #16704 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/MainConfigurationTest.php [hunk 1]
```diff
@@ -90,6 +90,28 @@ public function testCsrfAliases()
         $this->assertTrue(isset($processedConfig['firewalls']['stub']['logout']['csrf_token_id']));
         $this->assertEquals('a_token_id', $processedConfig['firewalls']['stub']['logout']['csrf_token_id']);
     }
+    public function testLegacyCsrfAliases()
```
Comment (Tobion) [inline src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/MainConfigurationTest.php]: missing newline

Comment (Tobion) [inline src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/MainConfigurationTest.php]: and I think we add the legacy annotation anyway

Comment (nicolas-grekas) [inline src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/MainConfigurationTest.php]: I agree with @tobion

Comment (xabbuh) [inline src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/MainConfigurationTest.php]: Yeah, may help us find it in the `master` branch.

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

**Pull request merged**